### ISSUE TITLE
PyInstaller 3.2, OneFile R6034, 32-bit Python 2.7.11

### ISSUE THREAD

**Issue by siliconplains44** [pyinstaller/pyinstaller#2042]
Some months back I built my exe with an older version of pyinstaller via a batch script i setup...

pyinstaller --windowed --clean  --icon icon.ico --onefile main.py --distpath c:\projects\datavaultg\dist\windows

Everything used to work great.

Now with pyinstaller 3.2 when I run the onefile exe I get the error R6034 Error in running the C Runtimes.

**Comment by stephenrauch** [comment]
The wiki has a [nice page on how to report bugs](https://github.com/pyinstaller/pyinstaller/wiki/How-to-Report-Bugs) so that the devs have the best chance of helping you. Having said that , this may be a duplicate of #1985.  That bug has been resolved on the development tip. From the wiki:

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_Please try latest development version and let us know if the bug is already fixed there. Simply use_

```
pip install https://github.com/pyinstaller/pyinstaller/archive/develop.zip
```

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;   _to install the latest git version._

That may get you going.  Please report back...

**Comment by spalato** [comment]
Hi,

I have encountered the same bug as the OP, in a one-dir mode. This is solved on the dev branch (3.3.dev0+a909f1a), both in one-dir and one-file modes.

I've reproduced it using a small script. See: 
https://groups.google.com/forum/?hl=en#!topic/pyinstaller/l9khDVLCfzI

**Comment by htgoebel** [comment]
I assume this is fixed with the current release of PyInstaller, since the bootloaders are now linked statically. If not, feel free to reopen the issue and provide more information.
